loadgen/loadgen.cc

```diff
@@ -313,6 +313,12 @@ std::vector<QueryMetadata> GenerateQueries(
   std::chrono::microseconds k2xTargetDuration = 2 * settings.target_duration;
   size_t min_queries = settings.min_query_count;
 
+  if (scenario == TestScenario::Server) {
+    min_queries_from_duration = DurationToSeconds(min_duration) * requested_qps;
```

Comment:
Since this is C++ now, you'll need to declare the type of the variable before using it:

min_queries_from_duration -> double min_queries_from_duration